docs/installation/autoware/docker-installation.md

```diff
@@ -77,7 +77,16 @@ You might need to log out and log back to make the current user able to use dock
    vcs import src < autoware.repos
    ```
 
-3. Build the workspace.
+3. Install missing dependencies.
+
```

Comment:
Thanks!

Can I add some description here? Please wait for a moment. :bow:

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/191#discussion_r962810050